docs/testing-guide.mdx

```diff
@@ -1,3 +1,7 @@
+import { Meta } from "@storybook/addon-docs";
```

Comment:
**comment** as with the `dev-environment-setup` file above, I think this is supposed to be a guide on Github and shouldn't be in our public-facing docs on storybook